lib/test/test_locust_api.py

```diff
@@ -15,15 +14,23 @@ def test_load_auth_from_config(self):
             self.assertEqual(api.password, 'halb')
 
     @mock.patch.dict('os.environ', {
-            LocustAuthHandler.LOCUST_USER: 'blah',
-            LocustAuthHandler.LOCUST_PASS: 'halb'
+            LocustAuthHandler.LOCUST_USER_FF: 'blah',
+            LocustAuthHandler.LOCUST_PASS_FF: 'halb'
     })
-    def test_load_auth_from_env(self):
-        LocustAuthHandler._verify_creds_in_env = MagicMock(return_value=True)
+    def test_load_auth_from_ff_env(self):
         api = LocustAuthHandler(use_env=True)
         self.assertEqual(api.username, 'blah')
         self.assertEqual(api.password, 'halb')
 
+    @mock.patch.dict('os.environ', {
+        LocustAuthHandler.LOCUST_USER_CGAP: 'blah',
```

Comment:
Are these actual passwords in the open? Should they not be passed as environment variables?